packages/shared/src/entities/activity-feed/activity.interface.ts

```diff
@@ -0,0 +1,30 @@
+import { ChannelTypeEnum } from '../../types';
+import { IExecutionDetail } from '../execution-details';
+import { IJob as IJobBase } from '../job';
+import { INotificationTemplate } from '../notification-template';
+import { ISubscriber } from '../subscriber';
+
+export interface IActivityJob extends IJobBase {
+  executionDetails: IExecutionDetail[];
+}
+
+export interface IActivity {
```

Comment:
We don't use `I` to mark the interface.

Reply:
This is located in the @novu/shared package, all of the interface here are with I prefix unless I'm missing something. For consistency across the package I will leave it as-is. We can do a complete refactor but it has to be across the board.

Reply:
For the new interfaces, we decided not to use the `I`, all the rest of the old interfaces will change with time, in small steps.